backend/src/main/java/com/redhat/cloud/notifications/db/repositories/EmailSubscriptionRepository.java

```diff
@@ -70,6 +78,7 @@ protected int replicateSubscribeToEventTypeLevel(String orgId, String username,
             .setParameter("applicationName", applicationName)
             .setParameter("bundleName", bundleName)
             .setParameter("subscriptionType", subscriptionType.name())
+            .setParameter("subscribed", true) // only email subscription replication is supported
```

Comment:
I think we should wrap the following code with an `if (emailSubscriptionType != DRAWER) { /* old code */ }`:

https://github.com/RedHatInsights/notifications-backend/blob/dd9ccdaf7ccd487e7bc30087f44ffcfe5d2e5d20/backend/src/main/java/com/redhat/cloud/notifications/routers/UserConfigResource.java#L104-L114